lms/djangoapps/commerce/signals.py

```diff
@@ -3,24 +3,16 @@
 """
```

Comment:
Can we also delete `handle_refund_entitlement` and the corresponding signal? I'd prefer to not have to maintain them if they're not being used.